pages/common/dolt-init.md

```diff
@@ -0,0 +1,12 @@
+# dolt init
+
+> Create an empty Dolt data repository.
+> More information: <https://docs.dolthub.com/interfaces/cli#dolt-config>.
+
+- Create an empty Dolt data repository in the current directory:
+
+`dolt init`
+
+- Specify details on the initial commit:
```

Comment:
Is this creating the first commit?

Reply:
this create a first commit of pure metadata in a .dolt directory

Reply:
```suggestion
- Initialize a new Dolt data repository creating a commit with the specified metadata:
```

Reply:
solved in 565108342595f9a73cdf474ac80034772f3aa5fb